src/helpers/script_helpers.sh

```diff
@@ -109,9 +109,15 @@ function run {
     undo_command
   else
     remove_step_files
+
+    # Running inside a subfolder can cause issues if that folder doesn't exist
+    # on the main branch. Hence we cd into the main branch here.
```

Comment:
Fixed the `cd` phrasing. The problem is indeed related to the folder not existing on the main branch. If it exists in both branches, no problem.